src/Aspire.Hosting.Azure.Storage/api/Aspire.Hosting.Azure.Storage.cs

```diff
@@ -8,10 +8,23 @@
 //------------------------------------------------------------------------------
 namespace Aspire.Hosting
 {
+    public partial class AzureBlobStorageContainerResource : ApplicationModel.Resource, ApplicationModel.IResourceWithConnectionString, ApplicationModel.IResource, ApplicationModel.IManifestExpressionProvider, ApplicationModel.IValueProvider, ApplicationModel.IValueWithReferences, ApplicationModel.IResourceWithParent<Azure.AzureBlobStorageResource>, ApplicationModel.IResourceWithParent
```

Comment:
This should be in the same namespace as the rest of the resource classes in this assembly.

Reply:
https://github.com/dotnet/aspire/pull/9285